modules/gdscript/gdscript_editor.cpp

```diff
@@ -1280,6 +1398,9 @@ static void _find_identifiers_in_base(const GDScriptCompletionIdentifier &p_base
 				ClassDB::get_enum_list(type, &enums);
 				for (const StringName &E : enums) {
 					int location = p_recursion_depth + _get_enum_location(type, E);
+					// Enum names for a native class, including `@GlobalScope`.
```

Comment:
Testing on recent dev I can't get any enum names from `@GlobalScope` to show at all so I'm not sure whether this is correct, and whether they should be handled here if it were to work.

Reply:
Thinking about it this would probably be the right place. Maybe we should add a TODO comment somewere or I'll create an issue for it.